### ISSUE TITLE
Batch size

### ISSUE THREAD

**Issue by InstantWindy** [facebookresearch/maskrcnn-benchmark#1114]
Hi. If I change the training batch size, how do the learning rate, steps, and max_ietr parameter change?  Thanks.

**Comment by Darshan2701** [comment]
@InstantWindy You need to check [https://github.com/facebookresearch/Detectron/blob/master/configs/getting_started/tutorial_1gpu_e2e_faster_rcnn_R-50-FPN.yaml#L14-L30
](url)
